pkg/asset/installconfig/openstack/openstack.go

```diff
@@ -67,37 +73,41 @@ func Platform() (*openstack.Platform, error) {
 			}),
 		},
 	}, &extNet)
-	if err != nil {
-		return nil, err
+	if extNet == noExtNet {
+		extNet = ""
 	}
-
-	floatingIPNames, err := validValuesFetcher.GetFloatingIPNames(cloud, extNet)
 	if err != nil {
 		return nil, err
 	}
-	sort.Strings(floatingIPNames)
-	var lbFloatingIP string
-	err = survey.Ask([]*survey.Question{
-		{
-			Prompt: &survey.Select{
-				Message: "APIFloatingIPAddress",
-				Help:    "The Floating IP address used for external access to the OpenShift API.",
-				Options: floatingIPNames,
+
+	lbFloatingIP := ""
```

Comment:
Dont set lbFloatingIP to empty, if it is not empty, this should be a validation error. It should be up to the user to not set it.